interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
The die part will be in the update-ns code itself. This is just a library routine. We don't panic() in any other code that was written for update-ns.